youtube_dl/postprocessor/embedthumbnail.py

```diff
@@ -89,10 +89,13 @@ def is_webp(path):
             os.rename(encodeFilename(temp_filename), encodeFilename(filename))
 
         elif info['ext'] in ['m4a', 'mp4']:
-            if not check_executable('AtomicParsley', ['-v']):
-                raise EmbedThumbnailPPError('AtomicParsley was not found. Please install.')
+            atomicparsley = 'AtomicParsley'
+            if not check_executable(atomicparsley, ['-v']):
+                atomicparsley = 'atomicparsley'
+                if not check_executable(atomicparsley, ['-v']):
```

Comment:
DRY.

Reply:
Cleaned it up, let me know if you have any other comments. Thx